moveit_ros/perception/mesh_filter/src/gl_renderer.cpp

```diff
@@ -387,7 +387,7 @@ void mesh_filter::GLRenderer::createGLContext()
 
   if (context_it == s_context.end())
   {
-    s_context.at(thread_id) = std::pair<unsigned, GLuint>(1, 0);
+    s_context[thread_id] = std::pair<unsigned, GLuint>(1, 0);
```

Comment:
I think we should explicitly call the insert method here because the thread_id doesn't exist if this scope is entered
```suggestion
    s_context.insert(thread_id) = std::pair<unsigned, GLuint>(1, 0);
```

Reply:
I tried it in my local workspace. It's okey. LGTM.